pkg/serverless/daemon/daemon.go

```diff
@@ -116,6 +116,7 @@ func StartDaemon(addr string) *Daemon {
 
 	mux.Handle("/lambda/hello", &Hello{daemon})
 	mux.Handle("/lambda/flush", &Flush{daemon})
+	mux.Handle("/trace-context", &TraceContext{})
```

Comment:
This endpoint is not prefixed by /lambda because we might want to reuse this endpoint for other cloud providers / other services and to prevent breaking changes in dd-trace-*